locales/english.json

```diff
@@ -1,61 +1,41 @@
   "search.view": "View",
   "search.lastUpdated": "Last updated on",
-  "page.authenticatedLoading": "Signing you in",
-  "page.unauthenticated": "Please sign in",
   "navbar.logo": "Equinor Data Marketplace",
-  "navbar.browse": "Browse",
-  "navbar.community": "Community",
   "navbar.placeholderSearch": "Search",
   "navbar.ariaUserMenu": "User menu",
-  "navbar.titleShoppingCart": "Shopping cart",
-  "navbar.taskStatistic": "Tasks: {remaining}",
-  "navbar.titleRefresh": "Refresh",
   "navbar.titleAccount": "Account",
-  "spinner.loading": "Please hold\u2026",
-  "asset.addToCart": "Add to cart",
   "asset.getAccess": "Get access",
   "asset.description": "Description",
   "asset.purpose": "Purpose",
   "asset.timeliness": "Timeliness",
   "asset.overview": "Overview",
   "asset.responsibilites": "Responsibilites",
-  "terms.ingress": "<p>These terms and conditions create a contract between you as a data consumer and the data provider (the “Agreement”).</p><p>When you are provided access to this Data Product, you will have to ensure that the following criteria are met:</p>",
+  "terms.ingress": "<p>The agreement between you and the data provider is created by these terms and use.</p><p>Access to this Data Product requires you to meet the following criteria.</p>",
   "terms.acceptLabel": "I accept the Terms of Use",
   "terms.accept.errorMessage": "You must accept the Terms of Use",
-  "terms.banner.danger.heading1": "Security Classification",
-  "terms.banner.danger.heading2": "Sharing",
-  "terms.banner.danger.description1": "Restricted",
-  "terms.banner.danger.description2": "Data from these Data Products can only be shared with restricted groups within Equinor. External sharing is prohibited.",
-  "checkout.access.headline": "Why do you want access?",
-  "checkout.access.ingress": "Please provide a short description explaining why you want access.",
   "checkout.access.exampleBody": "As a [role] I want [description of what you want to do with the data product], so that [description of why you need the data product]",
-  "checkout.access.descriptionInput.label": "Description (max. {maxLength} characters)",
-  "checkout.access.descriptionInput.placeholder": "Please provide a description\u2026",
+  "checkout.access.descriptionInput.label": "Why do you need access? (max. {maxLength} characters)",
   "checkout.access.descriptionInput.errorMessage": "This field is required and you must add at least {minLength} characters",
-  "checkout.redirect.headline": "Redirecting to AccessIT",
-  "checkout.redirect.ingress": "You'll now get redirected to AccessIT to complete the request process.",
-  "checkout.redirect.body": "If you don't get redirected within 15 seconds, please click this link: <a>{link}</a>",
+  "checkout.redirect.headline": "We're redirecting you to AccessIT to complete the request.",
+  "checkout.redirect.body": "If you’re not redirected within 10 seconds, follow this link: <a>{link}</a>",
```

Comment:
incorrect apostrophe. we should probably use unicode escapes to be consistent on special characters